src/website/app/demos/Flex/examples/Flex/alignItems.js

```diff
@@ -1,10 +1,11 @@
 /* @flow */
+import withProps from 'recompose/withProps';
 import { createStyledComponent } from '../../../../../../library/styles';
 import FlexItem from '../../components/FlexItem';
 import _DemoLayout from '../../components/DemoLayout';
 import _Flex from '../../components/Flex';
 
-const DemoLayout = _DemoLayout.withProps({ lastRowStartsAt: 5 });
```

Comment:
Ahh, this is probably why glamorous made it a method instead of an option. Good to know.

Reply:
Well, one could also just write the following...

```js
const DemoLayout = createStyledComponent(_DemoLayout, {}, { lastRowStartsAt: 5 });
```

Reply:
For internal or site-only work, either is fine by me. For public-facing code, we should only use `createStyledComponent`.

Reply:
For consistency, I just updated them all to use the `createStyledComponent` instead of `withProps`.